<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraManager</class>
 <widget class="QDialog" name="CameraManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>895</width>
    <height>332</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Camera editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="lstCameras">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="pageSelectCamera">
        <widget class="QLabel" name="lblSelectCameraToEdit">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>140</y>
           <width>631</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Select a camera to edit it</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageCameraImages">
        <widget class="QTabWidget" name="tabWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>661</width>
           <height>271</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabMainImage">
          <attribute name="title">
           <string>Main image</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="lblMainImage">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabFullMask">
          <attribute name="title">
           <string>Full mask</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="lblFullMask">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabHalfMask">
          <attribute name="title">
           <string>Half mask</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="lblHalfMask">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabFullWellMask">
          <attribute name="title">
           <string>Full well mask</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="lblFullWellMask">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabHalfWellMask">
          <attribute name="title">
           <string>Half well mask</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="lblHalfWellMask">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Export settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_640x480">
          <property name="text">
           <string>640x480</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_640x240">
          <property name="text">
           <string>640x240</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btnExportImage">
       <property name="text">
        <string>Export image</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btnDeleteCamera">
       <property name="text">
        <string>Delete selected camera</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btnSelectImage">
       <property name="text">
        <string>Choose image...</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btnDeleteImage">
       <property name="text">
        <string>Remove image</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="btnSetCameraId">
       <property name="text">
        <string>Set camera id</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QSpinBox" name="spnCameraId"/>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CameraManager</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
